modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -481,6 +488,39 @@ default. This record must be resolvable by both clients external to the cluster
 and from all the nodes within the cluster.
 |===
 
+
+[discrete]
+[id="installation-vsphere-installer-infra-static-ip-nodes_{context}"]
+== Specifying static IP addresses for vSphere nodes
+
+You can provision bootstrap, control plane, and compute nodes to use static IP addresses, so that you can better configure your network to meet the security demands of your organization. To configure this environment, you must provide values to the `platform.vsphere.hosts.role` parameter in the `install-config.yaml` file.
+
+:FeatureName: Specifying static IP addresses for vSphere nodes
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+By default, the installation program is configured to use the Dynamic Host Configuration Protocol (DHCP) for the network, but this network has limited configurable capabilities.
+
+The following example shows a `compute` network definition specified for the `role` parameter:
+
+[source,yaml]
+----
+---
+platform:
```

Comment:
SME query: Internal [writer guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#ip-addresses) state that only a certain range of IP addresses should be used in public-facing docs (depends on whether address are private or public). Are the addresses in the example public addresses?

![Screenshot from 2023-07-18 12-06-36](https://github.com/openshift/openshift-docs/assets/57954076/17fcdd63-ed0a-4de7-bcf8-8eda71647d36)